tests/main/views/test_buyers.py

```diff
@@ -1693,7 +1693,8 @@ def test_show_draft_brief_summary_page(self, data_api_client):
             ]
 
             assert "Awarded to " not in page_html
-            assert document.xpath('//a[contains(text(), "Delete")]')
+            assert document.xpath('//a[contains(text(), "Delete draft requirement")]')
+            assert not document.xpath('//a[contains(text(), "Withdraw requirement")]')
```

Comment:
We don;t need this `assert not` here. It should either be important enough to be a dedicated test or we shouldn't have it at all. `test_show_draft_brief_summary_page` is about things that _should_ be there.

Reply:
I was trying to be consistent with the assertions across all the status tests 😸 Agreed it's not ideal to assert the absence of something. How about instead we assert the exact list of all the links in that side div - there should only be one `column-one-third` div per overview page.